### PR TITLE
parallelizing the integration tests

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

**Comment by davidrabinowitz** [comment]
pre-submit cloud build job time has been reduced by 5 minutes

File: cloudbuild/cloudbuild.yaml [hunk 1]
```diff
@@ -4,36 +4,47 @@ steps:
     id: 'docker-build'
     args: ['build', '--tag=gcr.io/$PROJECT_ID/dataproc-spark-bigquery-connector-presubmit', '-f', 'cloudbuild/Dockerfile', '.']
 
-# 2. Fetch maven and dependencies
+  # 2. Fetch maven and dependencies
```
Comment (VinayLondhe14) [inline cloudbuild/cloudbuild.yaml:7]: Are these indentations intentional? If not, can we remove?

**Review by VinayLondhe14 — APPROVED** [review]

**Pull request merged**